### PR TITLE
Feature/display asap enabled

### PR DESCRIPTION
**Pull request by kkeeling**
Added support for enabling/disabling the "displayASAPEnabled" property for in-app messages

### CODE REVIEW

**Comment by kkeeling** [comment]
Sorry, pushed wrong commit

**Comment by kkeeling** [comment]
There, all set.

**Comment by hcrowell** [comment]
Thanks @kkeeling !
Could you fill out the contribution agreement please? http://docs.urbanairship.com/contribution-agreement.html 
We will review your PR once we receive your contribution agreement.
Thank you!

**Comment by kkeeling** [comment]
Done and done.

Keenan Keeling, Partner / Automata Studios
301.461.2163 / kkeeling@automatastudios.com / automatastudios.com
<http://www.automatastudios.com/>

**Comment by rlepinski** [comment]
Looks great

File: src/android/events/PushEvent.java [hunk 1]
```diff
@@ -63,6 +63,10 @@ public JSONObject getEventData() {
             if ("android.support.content.wakelockid".equals(key)) {
                 continue;
             }
+            if ("google.sent_time".equals(key)) {
```
Comment (rlepinski) [inline src/android/events/PushEvent.java]: nice, prevents the error? Could we had a new line between 65 & 66?

Comment (kkeeling) [inline src/android/events/PushEvent.java]: This prevents an error where sent_time cannot be converted from Long to String intrinsically in order to be added extras map. 

You want me to add a blank line between 65 and 66?

Comment (rlepinski) [inline src/android/events/PushEvent.java]: Its fine, I can add the line. Thanks!

**Comment by rlepinski** [comment]
Great job!

**Comment by kkeeling** [comment]
Thanks. 💃

**Pull request merged**